lib/linux/hci-ble.js

```diff
@@ -4,12 +4,13 @@ var events = require('events');
 var spawn = require('child_process').spawn;
 var util = require('util');
 
-var HciBle = function() {
-  var hciBle = __dirname + '/../../build/Release/hci-ble';
-
-  debug('hciBle = ' + hciBle);
+var HciBle = function(cfg) {
+  this._cfg = cfg
 
-  this._hciBle = spawn(hciBle);
+  var hciBle = __dirname + '/../../build/Release/hci-ble';
+  debug('hciBle = %s with hci%s', hciBle, this._cfg.NOBLE_HCI_DEVICE_ID);
+  var env = {NOBLE_HCI_DEVICE_ID : this._cfg.NOBLE_HCI_DEVICE_ID || '0' }
```

Comment:
I don't think it should default to 0, leave undefined, and let ```hci_get_route``` in the C layer chose the default.